kafka_topic-transferer/kafka-topic-transferer.go

```diff
@@ -0,0 +1,79 @@
+package transferer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	kf "github.com/codeuniversity/smag-mvp/kafka"
+	"github.com/codeuniversity/smag-mvp/models"
+	"github.com/codeuniversity/smag-mvp/utils"
+	"github.com/codeuniversity/smag-mvp/worker"
+
+	"github.com/segmentio/kafka-go"
+)
+
+// Transferer represents the Transferer containing all clients it uses
+type Transferer struct {
+	fromTopic *kafka.Reader
+	toTopic   *kafka.Writer
+
+	*worker.Worker
+}
+
+// New returns an initilized Transferer
+func New(fromTopic string, toTopic string) *Transferer {
+	t := &Transferer{}
+
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
+	groupID := utils.MustGetStringFromEnv("KAFKA_GROUPID")
+	readerConfig := kf.NewReaderConfig(kafkaAddress, groupID, fromTopic)
+	writerConfig := kf.NewWriterConfig(kafkaAddress, toTopic, true)
+
+	t.fromTopic = kf.NewReader(readerConfig)
+	t.toTopic = kf.NewWriter(writerConfig)
+
+	b := worker.Builder{}.WithName("kafka-topic-transferer").
+		WithWorkStep(t.runStep).
+		WithStopTimeout(10*time.Second).
+		AddShutdownHook("fromTopic", t.fromTopic.Close).
+		AddShutdownHook("toTopic", t.toTopic.Close)
+
+	t.Worker = b.MustBuild()
+
+	return t
+}
+
+//Run the Transferer
+func (t *Transferer) runStep() error {
+	fmt.Println("Start Transferer")
```

Comment:
This is also done by the worker
```suggestion
	fmt.Println("Start Transferer")
```